larq_compute_engine/mlir/ir/lce_ops.cc

```diff
@@ -1,10 +1,26 @@
 #include "larq_compute_engine/mlir/ir/lce_ops.h"
 
 #include "flatbuffers/flexbuffers.h"
+#include "tensorflow/lite/c/builtin_op_data.h"
 
 namespace mlir {
 namespace TF {
 
+static TfLitePadding ConvertTfLitePaddingAttr(llvm::StringRef str) {
+  return llvm::StringSwitch<TfLitePadding>(str)
+      .Case("SAME", kTfLitePaddingSame)
+      .Case("VALID", kTfLitePaddingValid);
```

Comment:
I am not 100% sure if this is the enum TFLite uses in the flatbuffer @Tombana  do you know?

Reply:
I think this is correct. The tflite conv2d kernel gets an `OpData` struct with a parameter `padding` which is directly passed into a function that expects these `kTfLitePaddingXXX` values.